README.md

```diff
@@ -110,6 +110,44 @@ if Mix.env == :dev do
 end
 ```
 
+## Disabling ansi_enabled setting
+
+The config `:ansi_enabled` is defaulted to `true` which will force the 
+console to have [ANSI coloring](https://hexdocs.pm/elixir/IO.ANSI.html#enabled?).  
+
+If Elixir can detect during startup that both `stdout` and `stderr` are terminals,
+then you set the value `:ignore`, for example shown below.
+
+```elixir
+# config/config.exs
+use Mix.Config
+
+if Mix.env == :dev do
+  config :mix_test_watch,
+    :ansi_enabled :ignore
+end
+```
+
+You might want to consider this setting when running tests that require a database. 
+The suggested approach is to create an alias in your mix.exs file, like
+
+```elixir
+  def aliases() do
+    ["test": ["ecto.drop --quiet", "ecto.create --quiet", "ecto.migrate", "test"]]
+  end
+```
+
+Unfortunately, when you run `test.watch` (without setting the `:ansi_enabled` value 
+to `:ignore`) you will probably observe the following error
```

Comment:
The `do run` appears to force the ecto supervisor to start, which holds on to conections, so the actual call of `ecto.drop --quiet` won't work as the DB is is use by others.

For example, 

```
$ mix test.watch

Running tests...
Compiling 6 files (.ex)
Generated stripe_callbacks app
** (Mix) The database for StripeCallbacks.Repo couldn't be dropped: 
ERROR 55006 (object_in_use): database "stripe_callbacks_test" 
is being accessed by other users

There are 10 other sessions using the database.
```